src/styles/blocks/content.css

```diff
@@ -231,15 +231,15 @@
     Основные стили задаются через блок media
 */
 
-.content img {
+.content img:not(.video__media) {
```

Comment:
```suggestion
.content > img {
```
А может лучше вот так?

Reply:
В принципе можно и так, но мне кажется это не очень наглядно. Просматривая классы, нельзя будет понять почему стили только для прямых потомков `.content`. И если где-нибудь будет просто вложенная картинка (не прямой потомок) то для нее стили уже не сработают. 
А так сразу понятно, что стили для всех картинок, кроме тех, что идут вместе с видео.

Reply:
Боюсь, что у нас картинки для видео — это не единственный пример, для которого нужно будет внедрить `:not(…)` и этот селектор разрастётся. По мне, так лучше описать в каких случаях он должен работать, чем сказать «всегда, кроме (и здесь большой список исключений)».

Reply:
Если будут еще исключение, тогда - да, лучше сделать как ты говоришь. Я такой вариант не рассмотрел.